chartpress.py

```diff
@@ -114,8 +115,15 @@ def _get_git_remote_url(git_repo):
     if not re.match(r"^[^/]+/[^/]+$", git_repo):
         return git_repo
 
+    github_actor = os.getenv(GITHUB_ACTOR_KEY)
     github_token = os.getenv(GITHUB_TOKEN_KEY)
-    if github_token:
+    if github_actor and github_token:
+        # this format works for a token created for a github
+        # workflow's job, no matter what we pass as github_actor.
+        return f"https://{github_actor}:{github_token}@github.com/{git_repo}"
+    elif github_token:
+        # this format works for personal access token, but
+        # not for a token created for a github workflow's job.
```

Comment:
Hmmm... Well... If someone use a personal access token in a github workflow, will this be breaking for them? I'll investigate.

Reply:
No! I've learned that a PAT works fine even if it includes some random content `before:` like this change could end up doing.

Reply:
In other words, this is not a breaking change even if a PAT is used in a github workflow.

(I also learned that PATs are recognized by being prefixed `ghp_` while the others are prefixed `ghs_`)